test/hooks.js

```diff
@@ -0,0 +1,28 @@
+import Color from "../src/index.js";
+import { check } from "./util.mjs";
+
+export default {
+	name: "Hooks Tests",
+	description: "TBD",
```

Comment:
```suggestion
```

It's not a mandatory field, so no need to have a description that just says "TBD".